core/main.go

```diff
@@ -246,6 +246,18 @@ func main() {
 
```

Comment:
A general structural question -- What does the "core" package signify. Is that the same as the "cmd" package in all other services? If so, please rename.

Reply:
It has different meaning due to the historical reason. The secret-service was implemented with bash script in the first version and the core was create to hold the golang implementation in the follow version. Would prefer to keep the name of the folder at the last minute unless it is absolute needed. we can open an issue regarding this and put it into backlog to address in the next release cycle.

Reply:
project has been reorganized.